addons/website_forum/controllers/main.py

```diff
@@ -341,7 +341,7 @@ def forum_post(self, forum, post_type=None, **post):
     def post_create(self, forum, post_parent=None, post_type=None, **post):
         if post_type == 'question' and not post.get('post_name', ''):
             return request.render('website.http_error', {'status_code': _('Bad Request'), 'status_message': _('Title should not be empty.')})
-        if post.get('content', '') == '<p><br></p>':
+        if post.get('content', '') == '<p class="fa fa-user"><br></p>':
```

Comment:
???

'<p><br></p>' is default content of an empty message...

Where this fa-user is comming from?